### PR TITLE
Update linting documentation and workflow examples

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: developer_manual/digging_deeper/continuous_integration.rst [hunk 1]
```diff
@@ -43,28 +45,33 @@ You can validate the ``info.xml`` :ref:`app metadata<app metadata>` file of an a
...
+  {
+    "scripts": {
+      "lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l"
+    }
+  }
+
+php-cs
+^^^^^^
+
+We encourage the usage of php-cs linting. You will need the ``.php_cs.dist`` config file at the root of your repository.
```
Comment (ChristophWurst) [inline developer_manual/digging_deeper/continuous_integration.rst:62]: Let's just link to https://github.com/nextcloud/coding-standard. That covers the setup steps.

Comment (skjnldsv) [reply developer_manual/digging_deeper/continuous_integration.rst]: Done

File: developer_manual/digging_deeper/continuous_integration.rst [hunk 2]
```diff
@@ -43,28 +45,34 @@ You can validate the ``info.xml`` :ref:`app metadata<app metadata>` file of an a
...
+You will also require the following scripts in your ``composer.json``:
+
+.. code-block:: json
+
+  {
+    "scripts": {
+      "cs:fix": "php-cs-fixer fix",
+      "cs:check": "php-cs-fixer fix --dry-run --diff"
+    }
+  }
```
Comment (skjnldsv) [inline developer_manual/digging_deeper/continuous_integration.rst:74]: Do you also want me to remove this @ChristophWurst ?

Comment (ChristophWurst) [reply developer_manual/digging_deeper/continuous_integration.rst:74]: yes, let's only have this at one place – the readme :+1:

Comment (skjnldsv) [reply developer_manual/digging_deeper/continuous_integration.rst]: Done! :rocket:

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**